crates/ruff/src/resolver.rs

```diff
@@ -128,7 +128,7 @@ pub fn resolve_configuration(
 
         // Resolve the current path.
         let options = pyproject::load_options(&path)
-            .map_err(|err| anyhow!("Failed to parse `{}`: {}", path.to_string_lossy(), err))?;
+            .map_err(|err| anyhow!("Failed to parse `{}`: {}", path.display(), err))?;
```

Comment:
Separately: pretty sure we want to be using `.display` when constructing user-facing paths?